docs/content/configuration.md

```diff
@@ -283,7 +283,53 @@ Following successful authentication at the token endpoint the returned token wil
 | `services[_].credentials.oauth2.client_secret` | `string` | Yes | The client secret to use for authentication. |
 | `services[_].credentials.oauth2.scopes` | `[]string` | No | Optional list of scopes to request for the token. |
 
-#### AWS signature
+#### OAuth2 JWT Bearer Grant Type
+
+OPA will authenticate using a bearer token obtained through the OAuth2 [JWT authorization grant](https://tools.ietf.org/html/rfc7523#section-2.1) flow.
+Rather than providing a client secret along with the request for an access token, the client [asserts](https://tools.ietf.org/html/rfc7521#section-4.1) its identity in the form of a signed JWT.
+
+| Field | Type | Required | Description |
+| --- | --- | --- | --- |
+| `services[_].credentials.oauth2.token_url` | `string` | Yes | URL pointing to the token endpoint at the OAuth2 authorization server. |
+| `services[_].credentials.oauth2.grant_type` | `string` | Yes | Must be set to `jwt_bearer`. |
```

Comment:
Not accurate anymore, I guess

Reply:
Well it is accurate with regards to the JWT bearer flow, which is what is documented here, but I should probably rephrase that.

Reply:
Ah sorry my bad. Yeah that makes sense, I was thinking this was a general section for the allowed inputs but it's not. 👍